src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx

```diff
@@ -2104,9 +2171,206 @@ public App ()
                 ```
             </td>
         </tr>
+          <tr>
+            <td>
+                Enable or disable offline data storage when no internet connection is available. To configure the amount of offline storage, see [Set max offline storage size](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-offline-storage/#xamarin).
+            </td>
+            <td>
+                Offline storage is enabled by default. To disable it, add the following feature flag:
+
+                ```typescript
+                offlineStorageEnabled:false
+                ```
+            </td>
+        </tr>
     </tbody>
 </table>
 
              </TabsPageItem>
+
+             <TabsPageItem id="unity">
```

Comment:
@ndesai-newrelic We're missing a whole tab for unity on the configure settings page. I copied this over from another hybrid agent, but I need you to verify if the code samples are correct for Unity.

Reply:
user can configure using ui, we just need to put UI screenshot for that.

Reply:
<img width="955" alt="Screenshot 2024-03-13 at 1 03 45 PM" src="https://github.com/newrelic/docs-website/assets/89222514/fae0151e-cd64-4470-b467-3b8304adca0e">